src/cli/tsconfig.ts

```diff
@@ -58,13 +58,33 @@ export function parseConfigFileWithSystem(
     return updateParsedConfigFile(parsedConfigFile);
 }
 
+function resolveModuleConfig(
+    moduleName: string,
+    configRootDir: string,
+    host: ts.ModuleResolutionHost
+): string | undefined {
+    const resolved = ts.nodeNextJsonConfigResolver(moduleName, path.join(configRootDir, "tsconfig.json"), host);
+    if (resolved.resolvedModule) {
+        return resolved.resolvedModule.resolvedFileName;
+    }
+}
+
 function getExtendedTstlOptions(
     configFilePath: string,
     configRootDir: string,
     cycleCache: Set<string>,
     system: ts.System
 ): TypeScriptToLuaOptions {
-    const absolutePath = path.isAbsolute(configFilePath) ? configFilePath : path.resolve(configRootDir, configFilePath);
+    const absolutePath = ts.pathIsAbsolute(configFilePath)
+        ? configFilePath
+        : ts.pathIsRelative(configFilePath)
+        ? path.resolve(configRootDir, configFilePath)
+        : resolveModuleConfig(configFilePath, configRootDir, system);
```

Comment:
I would still like you to add a comment in code explaining that if a path is not absolute and not relative that it is then a module path that needs to be resolved